README.MD

```diff
@@ -167,23 +235,36 @@ default: &default
   password:
 ```
 
```

Comment:
Need to add:
for epp-proxy:
```
%% Path to root CA that should check the client certificates.
{cacertfile_path, "/opt/ca/certs/ca.crt.pem"},

```
For registry application.yml (this is mandatory to registrar portal to work):
`webclient_ips: '127.0.0.1'`

Reply:
Added.